pandas-stubs/core/indexing.pyi

```diff
@@ -83,6 +83,18 @@ class _LocIndexerFrame:
         value: Union[DataFrame, Series, float],
     ) -> None: ...
 
+class _AtIndexerFrame:
+    # get item
+    def __getitem__(self, idx: Tuple[int, _StrLike]) -> float: ...
+    # set item
+    def __setitem__(self, idx: Tuple[int, _StrLike], value: float) -> None: ...
+
+class _AtIndexerSeries:
```

Comment:
For the Series object, the situation is easier: it can only have one type. So we'll make this generic in the dtype:
```suggestion
class _AtIndexerSeries(Generic[_DType]):
```